mono/mini/mini-llvm.c

```diff
@@ -3216,7 +3216,12 @@ emit_gc_safepoint_poll (MonoLLVMModule *module, LLVMModuleRef lmodule, MonoCompi
 	LLVMValueRef func = mono_llvm_get_or_insert_gc_safepoint_poll (lmodule);
 	mono_llvm_add_func_attr (func, LLVM_ATTR_NO_UNWIND);
 	if (is_aot) {
-		LLVMSetLinkage (func, LLVMWeakODRLinkage);
+#if TARGET_WIN32
```

Comment:
Not sure what  LLVMWeakODRLinkage was trying to achieve here.

Reply:
me neither, only thing I know is that it will fail under msvc when doing static link due to duplicated symbols in obj's.

Reply:
are you leaning towards do the same for static linking on all platforms not just Windows?